src/NewCommand.php

```diff
@@ -243,4 +283,73 @@ protected function findComposer()
 
         return 'composer';
     }
+
+    /**
+     * @param  string  $packages
+     * @param  string  $composer
+     * @return  string
+     */
+    private function getLaravelPackages($packages, $composer)
```

Comment:
This should be `protected` instead.

Reply:
The only difference between `getLaravelPackages` and `getLaravelDevPackages` is the flag `--dev` and the `list of packages`.

Maybe consider making a generic function like `getPackages($composer , $with, $dev = true)`.